pkg/apb/rhcc_registry.go

```diff
@@ -66,10 +67,10 @@ func (r RHCCRegistry) LoadSpecs() ([]*Spec, int, error) {
 	r.log.Debug("Found %d images in RHCC", numResults)
 	for _, image := range imageList.Results {
 		spec, err := r.imageToSpec(image)
-		if err != nil {
-			return []*Spec{}, 0, err
+		// Want to ignore errors here because some images will not be an APB
+		if err == nil {
+			specs = append(specs, spec)
 		}
```

Comment:
This gives me an error because `ok` is a non-bool (type error). I have what you put here but instead kept it as `err` (since ok being nil implies not ok) and said err == nil.